css/src/components/table.css

```diff
@@ -0,0 +1,168 @@
+.yoast table {
+	font-size: var(--font-size-default);
+	color: var(--color-default);
+	line-height: 1.2;
```

Comment:
This is correct: line-height values should always be unit less. However, there's another line-height value in this file that uses pixels (incorrect and inconsistent).